src/metatrain/utils/metrics.py

```diff
@@ -8,9 +8,10 @@ class RMSEAccumulator:
     """Accumulates the RMSE between predictions and targets for an arbitrary
     number of keys, each corresponding to one target."""
 
-    def __init__(self) -> None:
+    def __init__(self, separate_blocks: bool = False) -> None:
```

Comment:
Some docs for `separate_blocks` here for developers who want to use the class? Should be fairly self explanatory, but removes some guess work